src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -423,6 +432,16 @@ public Result addResultToSubmission(Submission submission) {
         return result;
     }
 
+    public Set<GradingInstruction> addGradingInstructionsToExercise(Exercise exercise) {
```

Comment:
Maybe add the amount of instructions to add as a parameter

Reply:
Could you please explain why I should add the number of instructions?

Reply:
So that developers that read through code, which calls this method, immediately know how many instructions get created and so that you can be flexible in future tests (maybe you will have some tests that require only one instruction, maybe more, etc.)